cmd/genericconf/config.go

```diff
@@ -65,6 +65,27 @@ var DefaultS3Config = S3Config{
 	SecretKey: "",
 }
 
+type GoogleCloudStorageConfig struct {
+	AccessToken  string        `koanf:"access-token"`
```

Comment:
The pattern for the other storage backend types is to have an `Enable bool` field, defaulting to false.

Reply:
Actually it looks like you've got duplicated structs that do almost the same thing. `GoogleCloudStorageServiceConfig` is the other one.

Reply:
Sorry but the is one for unit tests only? also defaultStorageRetention not exported from das, so cannot use it here(genericconf)

Reply:
I was referring the s3 part, so GoogleCloudStorageConfig for ut, GoogleCloudStorageServiceConfig is for actual usage, hope i didn't get it wrongly : D

Reply:
while GoogleCloudStorageServiceConfig follow the same pattern

```
type GoogleCloudStorageServiceConfig struct {
	Enable       bool          `koanf:"enable"`
	AccessToken  string        `koanf:"access-token"`
	Bucket       string        `koanf:"bucket"`
	ObjectPrefix string        `koanf:"object-prefix"`
	EnableExpiry bool          `koanf:"enable-expiry"`
	MaxRetention time.Duration `koanf:"max-retention"`
}
```

Reply:
I think you can remove this one and just use `das.GoogleCloudStorageServiceConfig` for the `das` unit tests since they're in the same package.